sdk/src/beta9/config.py

```diff
@@ -73,10 +76,26 @@ def is_valid(self) -> bool:
 
 
 def set_settings(s: Optional[SDKSettings] = None) -> None:
-    if s is None:
+    global _SETTINGS
+
+    if env.is_local() and s is None:
         s = SDKSettings()
 
-    global _SETTINGS
+        if "beam" in sys.modules:
+
+            @dataclass
+            class Settings(SDKSettings):
+                realtime_host: str = os.getenv("REALTIME_HOST", "wss://rt.beam.cloud")
+
+            s = Settings(
+                name="Beam",
+                api_host=os.getenv("API_HOST", "https://app.beam.cloud"),
+                gateway_host=os.getenv("GATEWAY_HOST", "gateway.beam.cloud"),
+                gateway_port=int(os.getenv("GATEWAY_PORT", 443)),
+                config_path=Path("~/.beam/config.ini").expanduser(),
+                use_defaults_in_prompt=True,
+            )
+
     _SETTINGS = s
```

Comment:
When is _SETTINGS set if this is remote? Does it work properly?

Reply:
The logic here is the same as it was when remote. If called without an explicitly passed in settings it will just load the default. I didn't want to change the remote behavior, which is why I included the local check.

Reply:
I found that the import of `sync.py` was causing the default init of the settings and changed that instead of doing the module checking.